espressif/vscode-esp-idf-extension
# JTAG Flash Support (VSC-487)

This feature will enable to flash device via JTAG

##### TODOs:

- [x] Add `i18n`
- [x] Restrict simultaneous flash via `UART` & `JTAG`


Closes #161 

---

##### How to test?

- Set the settings `idf.flashType` as `JTAG` and click flash button and it should use the functionality for flashing via JTAG

---

##### Developers Note

> OpenOCD Version Requirements: [v0.10.0-esp32-20201125](https://github.com/espressif/openocd-esp32/releases/tag/v0.10.0-esp32-20201125) and above

> Awaits release and backport to IDF
